code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -958,7 +958,7 @@ nobiliumsuppression = INFINITY
 
 		var/new_heat_capacity = air.heat_capacity()
 		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
-			air.temperature = (temperature * old_heat_capacity + energy_released) / new_heat_capacity
+			air.temperature = max(((temperature * old_heat_capacity + energy_released) / new_heat_capacity), TCMB)
```

Comment:
You don't need the second () there, in fact it clutters the code a bit
```suggestion
			air.temperature = max((temperature * old_heat_capacity + energy_released) / new_heat_capacity, TCMB)
```
Most all of the maxes have this issue